packages/cli/src/options/common.js

```diff
@@ -44,5 +44,10 @@ export default {
     alias: 'h',
     type: 'boolean',
     description: 'Display this message'
+  },
+  'no-loading-screen': {
```

Comment:
Adding a CLI flag for a special requirement is unnecessary. With supporting `build.indicator` option, one can disable it based on environment variables from `nuxt.config`.